next/MultipleContents/README.md

```diff
@@ -0,0 +1,79 @@
+# Multiple Contents
+
+This sample demonstrates usage of the [`3DTILES_multiple_contents`](https://github.com/CesiumGS/3d-tiles/tree/main/extensions/3DTILES_multiple_contents) extension for storing multiple contents objects in a single tile.
+
+![MultipleContents](screenshot/MultipleContents.gif)
+
+The root tile of the tileset contains a model, namely a simple plane with sine waves, in two different representations:
+
+- A low-resolution representation as a textured triangle mesh glTF asset
+- A high-resolution point-grid representation with vertex colors, as another glTF asset
+
+The following sandcastle code renders this example tileset when it is hosted on a local server, and allows toggling the visibility of both representations:
+
+```
+// Enable experimental features for 3DTILES_multiple_contents support
+Cesium.ExperimentalFeatures.enableModelExperimental = true;
+
+var viewer = new Cesium.Viewer('cesiumContainer');
+
+// Flags that are connected to the Sandcastle toggle buttons,
+// to show and hide the triangle- and point based content
+var showTriangles = true;
+var showPoints = true;
+
+// The two different contents from the root tile. These are
+// read in the 'tileLoad' event listener, and stored here
+// for access from the Sandcastle UI
+var trianglesContent;
+var pointsContent;
+
+// Load the tileset from a local server
+var tileset = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
+    url : 'http://localhost:8003/tileset.json'
+}));
+
+// Attach the listener that will be notified when a tile is loaded.
+// It will store the 'trianglesContent' and 'pointsContent' from
+// the root tile, for later access
+tileset.tileLoad.addEventListener(function(tile) {
+  var content = tile.content;
+  var innerContents = content.innerContents;
+  if (Cesium.defined(innerContents) && innerContents.length === 2) {
+    trianglesContent = innerContents[0];
+    pointsContent = innerContents[1];
+  }
+});
+
+// Wait until the tileset is loaded, move it to a certain position
+// on the globe, and center the tileset in the viewer
+tileset.readyPromise.then(function() {
+  tileset.modelMatrix = Cesium.Transforms.eastNorthUpToFixedFrame(
+    Cesium.Cartographic.toCartesian(
+      Cesium.Cartographic.fromDegrees(-75.1596759, 39.9509025, 0)));
+  viewer.zoomTo(tileset);
```

Comment:
You don't need to wait for the ready promise, `zoomTo()` already does this, see https://cesium.com/learn/cesiumjs/ref-doc/Viewer.html?classFilter=viewer#zoomTo